code/modules/reagents/reagent_containers/food/snacks.dm

```diff
@@ -32,11 +32,17 @@
 /obj/item/reagent_containers/food/snacks/on_consume(mob/user, mob/target)
 	if(!reagents.total_volume && !trash)
 		target.visible_message("<b>[target]</b> finishes [is_liquid ? "drinking" : "eating"] \a [src].",
-					 SPAN_NOTICE("You finish [is_liquid ? "drinking" : "eating"] \a [src]."))
+				   SPAN_NOTICE("You finish [is_liquid ? "drinking" : "eating"] \a [src]."))
+		if(src.reagents_to_add == bitesize)
+			target.visible_message("<b>[target]</b> finishes [is_liquid ? "drinking" : "eating"] \a [src].",
```

Comment:
isn't this going to make two visible messages

Reply:
oops